src/redux/toDoSlice/toDoSlice.js

```diff
@@ -0,0 +1,42 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const toDoSlice = createSlice({
+  name: "toDo",
+  initialState: {
+    toDoList: [],
+  },
+  reducers: {
+    startedApp: (state) => {
```

Comment:
Is there any specific reason to have this reducer? i mean you have just put a console.log().
Use can use that for your own testing and understanding but avoid pushing unnecessary code.

Reply:
I used this as a trigger for a saga generator function, it's called only once in the very beginning, saga catches it and loads previous to-dos from the db into redux